docs/sql-reference/sql-statements/data-definition/DROP TABLE.md

```diff
@@ -1,21 +1,21 @@
 # DROP TABLE
 
-## description
+## Description
```

Comment:
Shell we modify those format things in such unfied ways?
I see that a lot of other pages haven't been modified yet.

And, will we remove the "Keyword" section? (I think it's good to keep it)

Reply:
We will gradually modify the format of titles. "The Keyword section(in SQL and function) needed to be deleted. " It already confirmed with simo.